base/ipmicomm.patch

```diff
@@ -1,5 +1,24 @@
 From: https://github.com/slac-epics-modules/ipmiComm/pull/26/
+From: https://github.com/slac-epics-modules/ipmiComm/pull/27/
```

Comment:
I think you wanted to reference https://github.com/slac-epics-modules/ipmiComm/pull/29 instead.